src/constants.ts

```diff
@@ -6,8 +6,10 @@ export const IPC_EVENTS = {
 	COMPRESS_IMAGE_FAIL: 'imageOptimizer:image-compress-fail',
 	COMPRESS_ALL_IMAGES_COMPLETE: 'imageOptimizer:image-compress-all-complete',
 	GET_IMAGE_DATA: 'imageOptimizer:get-image-data',
+	GET_IMAGE_DATA_STORE: 'imageOptimizer:get-image-data-store',
```

Comment:
Seems like we have two naming conventions going on here. Might be good to go with one and stick with it.

Reply:
Good callout 🙏